labs/bbb-webrtc-sfu/config/default.example.yml

```diff
@@ -18,9 +18,13 @@ to-akka: "to-akka-apps-redis-channel"
 from-akka: "from-akka-apps-redis-channel"
 common-message-version: "2.x"
 webcam-force-h264: true
+# Target bitrate (kbps) for webcams. Value 0 leaves it unconstrained.
```

Comment:
Are the units here `bps` or `kbps`?
I think it should be either 300 kbs or 300 000 bps

Same for the unit on ` screenshare-target-bitrate`

Reply:
It's `kbps`. Sorry, I changed it from `bps` to `kbps` but forgot to update the example file. Will update it.

Reply:
fixed